### PR TITLE
Add default configs to Console

### PR DESCRIPTION
**Pull request by savindi7**
- Approvals
- Attributes
- Certificates
- Email templates
- Governance connectors
- Scopes
- Secrets management
- Remote configurations
- User stores

>All features other than `Approvals` will be grouped into one configuration to enable/disable them.
>The facility to enable/disable `Approvals` seperately is already available.

>The default views of the Console will be as follows.

-Develop tab

Screenshot 2022-07-19 at 12 16 17

-Manage tab

Screenshot 2022-07-19 at 12 15 37

issue - https://github.com/wso2/product-is/issues/14205

#### Documentation

>Relevant documentation will need to be updated to inform users of the new group config. 
>In deployment.toml file the following configuration will enable all the features included in the group config.

`[console]`
`additional_features.enabled=true`

-Develop tab (with additional features enabled)

Screenshot 2022-07-19 at 12 26 40

-Manage tab (with additional features enabled)

Screenshot 2022-07-19 at 12 27 29

>Approvals or other features can be enabled seperately as follows.

`[console]`
`approvals.enabled=true`

-Manage tab (with additional features and approvals enabled)

Screenshot 2022-07-19 at 12 36 01

### CODE REVIEW

**Review by yasinmiran — APPROVED** [review]

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json [hunk 1]
```diff
@@ -680,7 +680,7 @@
   "console.session.params.sessionRefreshTimeOut": 300,
   "console.session.params.checkSessionInterval": 3,
   "console.applications.enabled": true,
-  "console.product_version.configs.productVersion": "BETA",
+  "console.product_version.configs.productVersion": "",
```
Comment (piraveena) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:683]: do we need this config here? we can remove it from the default.json file

Comment (savindi7) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:683]: This commit removes it https://github.com/wso2/carbon-identity-framework/pull/4129/commits/d1160f2b108a0610d09266b80ec89044d1f4a3ba

Comment (piraveena) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:683]: we don't need this from config in the default.json file.

Comment (savindi7) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json]: Fixed with https://github.com/wso2/carbon-identity-framework/pull/4129/commits/ad2dd6a4a862977397856428e1c3f71d4646c784

**Review by piraveena — APPROVED** [review]

**Pull request merged**